### PR TITLE
Add release note 1.5.0

### PR DESCRIPTION
**Pull request by chunseoklee**
- This is a temaplate of release note 1.5.0

ONE-DCO-1.0-Signed-off-by: Chunseok Lee <chunseok.lee@samsung.com>

### CODE REVIEW

File: docs/release/1.5/release-note-1.5.0.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+- `ONE Compiler`
+    - 
+
+- `ONE Runtime`
+    -
+
+## ONE Compiler
+
+## ONE Runtime
+
```
Comment (hseok-oh) [inline docs/release/1.5/release-note-1.5.0.md:14]: #### CPU backend supports more operations
The following operations are supported on CPU backend :

- ArgMax
- Cos
- ExpandDims
- Fill
- Log
- LogicalNot
- LogicalOr
- Mean
- Neg
- Pow
- ReLU
- ReduceAny
- ReduceProd
- Reverse
- Round
- Select
- SquaredDifference
- Tile
- ZerosLike

File: docs/release/1.5/release-note-1.5.0.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Release Note 1.5.0
+
+## Feature Highlights
+
+- `ONE Compiler`
+    - 
+
+- `ONE Runtime`
+    -
```
Comment (hseok-oh) [inline docs/release/1.5/release-note-1.5.0.md:9]: - CPU backend supports more operations

File: docs/release/1.5/release-note-1.5.0.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Release Note 1.5.0
+
+## Feature Highlights
+
+- `ONE Compiler`
```
Comment (hyunsik-yoon) [inline docs/release/1.5/release-note-1.5.0.md:5]: @lemmaa created several PR to change `ONE` or _ONE_ to **ONE**. Maybe we need to use "**ONE** compiler"?

**Comment by seanshpark** [comment]
OP list of compiler (`tflite2circle` and `circle2circle`)

Abs
Add
ArgMax
AvgPool2D
BatchToSpaceND
Cast
Concat
Const
Conv2D
Cos
Custom
DepthwiseConv2D
Div
Elu
Equal
Exp
ExpandDims
Fill
FloorDiv
FloorMod
FullyConnected
Gather
GatherNd
Greater
GreaterEqual
If
LeakyRelu
Less
LocalResponseNormalize
LogicalAnd
LogicalNot
LogicalOr
Logistic
Maximum
MaxPool2D
Mean
Minimum
MirrorPad
Mul
Neg
NotEqual
OneHot
Pack
Pad
Pow
Range
ReduceAny(Any)
ReduceMax(Max)
ReduceProd
ReduceSum(Sum)
ReLU
RELU_N1_TO_1
Reshape
ResizeNearestneighbor
Rsqrt
Select
Shape
Sin
Slice
Softmax
SpaceToBatchND
SpaceToDepth
Split
SplitV
Square
SquaredDifference
Squeeze
StridedSlice
Sub
Tanh
Tile
TopKV2
Transpose
Unpack(Unstack)
While
ZerosLike

File: docs/release/1.5/release-note-1.5.0.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    - Compiler supports more operations
+
+- `ONE Runtime`
+    - CPU backend supports more operations
+
+## ONE Compiler
+
+### Compiler supports more operations
+
+The following operations are supported on CPU backend :
```
Comment (llFreetimell) [inline docs/release/1.5/release-note-1.5.0.md:15]: How about removing `CPU backend` ?

Comment (chunseoklee) [reply docs/release/1.5/release-note-1.5.0.md:15]: nice catch

File: docs/release/1.5/release-note-1.5.0.md (hunk 3 above)
Comment (hyunsik-yoon) [reply docs/release/1.5/release-note-1.5.0.md:5]: I've heard not answer but it was marked _resolved_.
Sorry but I worked in marketing before so I am obsessed with naming and how it is shown to public.

**Comment by hseok-oh** [comment]
@hyunsik-yoon PTAL

**Comment by hyunsik-yoon** [comment]
@hseok-oh @chunseoklee I left a comment: https://github.com/Samsung/ONE/pull/1462#discussion_r432270405

File: docs/release/1.5/release-note-1.5.0.md (hunk 3 above)
Comment (chunseoklee) [reply docs/release/1.5/release-note-1.5.0.md:5]: This is my mistake. I will update.

Comment (chunseoklee) [reply docs/release/1.5/release-note-1.5.0.md:5]: @hyunsik-yoon I have bolded ONE word.

File: docs/release/1.5/release-note-1.5.0.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+- Abs, Add, ArgMax, AvgPool2D, BatchToSpaceND, Cast, Concat, Const, Conv2D, Cos,
+Custom, DepthwiseConv2D, Div, Elu, Equal, Exp, ExpandDims, Fill, FloorDiv,
+FloorMod, FullyConnected, Gather, GatherNd, Greater, GreaterEqual, If,
+LeakyRelu, Less, LocalResponseNormalize, LogicalAnd, LogicalNot, LogicalOr,
+Logistic, Maximum, MaxPool2D, Mean, Minimum, MirrorPad, Mul, Neg, NotEqual,
+OneHot, Pack, Pad, Pow, Range, ReduceAny(Any), ReduceMax(Max), ReduceProd,
+ReduceSum(Sum), ReLU, RELU_N1_TO_1, Reshape, ResizeNearestneighbor, Rsqrt,
+Select, Shape, Sin, Slice, Softmax, SpaceToBatchND, SpaceToDepth, Split, SplitV,
+Square, SquaredDifference, Squeeze, StridedSlice, Sub, Tanh, Tile, TopKV2,
+Transpose, Unpack(Unstack), While, , ZerosLike
```
Comment (lemmaa) [inline docs/release/1.5/release-note-1.5.0.md:26]:
```suggestion
Transpose, Unpack(Unstack), While, ZerosLike
```

Duplicate comma.

**Pull request merged**